pkg/server/datastore/sqldriver/awsrds/awsrds.go

```diff
@@ -0,0 +1,184 @@
+package awsrds
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/go-sql-driver/mysql"
+	"github.com/jackc/pgx/v5"
+	"github.com/jinzhu/gorm"
+	"github.com/lib/pq"
+)
+
+const (
+	MySQLDriverName    = "aws-rds-mysql"
+	PostgresDriverName = "aws-rds-postgres"
+	iso8601BasicFormat = "20060102T150405Z"
+	timeOut            = time.Second * 30
```

Comment:
```suggestion
	connectionTimeout            = time.Second * 30
```

Reply:
This timeout is not a connection timeout to the database, but a timeout building the authentication token. Maybe `buildAuthTokenTimeout` is a better name.